release notes/v0.49.0.md

```diff
@@ -0,0 +1,175 @@
+k6 `v0.49.0` is here 🎉! This release includes:
+
+- clear to the locator classes for the browser module
```

Comment:
```suggestion
- clear the locator classes for the browser module
```

Reply:
I think that you suggestion makes it seem like the locator class was cleared of something.

While in reality it is adding a `clear` method to it - that is explained below.

On the other hand the current strucutre is also not great :( 

```suggestion
- add `clear` functionality to the `locator` classes within the browser module
```

?

Reply:
Indeed!

Your suggestion is a great improvement indeed 👍🏻